### PR TITLE
Butter_HPC

### PR DESCRIPTION
**Pull request by MichaldotLevin**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

**Comment by MichaldotLevin** [comment]
I reviewed the text and it should be fine now.

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please

File: nfcore_custom.config [hunk 1]
```diff
@@ -20,6 +20,7 @@ profiles {
     bigpurple      { includeConfig "${params.custom_config_base}/conf/bigpurple.config" }
     binac          { includeConfig "${params.custom_config_base}/conf/binac.config" }
     biohpc_gen     { includeConfig "${params.custom_config_base}/conf/biohpc_gen.config" }
+    BUTTER_HPC     { includeConfig "${params.custom_config_base}/conf/BUTTER_HPC.config" }
```
Comment (maxulysse) [inline nfcore_custom.config:23]: Can the profile be lowercased, and so for the file name?

Comment (MichaldotLevin) [reply nfcore_custom.config:23]: Yes I will take care of that and let you know when finished.

Comment (MichaldotLevin) [reply nfcore_custom.config:23]: Done lowercasing all relevant file names and contents!

Comment (maxulysse) [reply nfcore_custom.config:23]: still hasn't changed

**Comment by maxulysse** [comment]
@MichaldotLevin any update?
There are still some changes needed to be done before we can merge

File: nfcore_custom.config (hunk 1 above)
Comment (MichaldotLevin) [reply nfcore_custom.config]: I took care of this now

**Comment by MichaldotLevin** [comment]
> @MichaldotLevin any update? There are still some changes needed to be done before we can merge

**Comment by MichaldotLevin** [comment]
Dear Maxime, can we try the merge again? I think to have accomplished all open issues now.

**Comment by MichaldotLevin** [comment]
Can we check this again?

**Pull request closed without merging**